docker/build.sh

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env bash
+
+set -euxo pipefail
```

Comment:
I'm not sure all flags are supported by all shells, but let's ignore that for now.

Reply:
`dash` doesn't support `pipefail`, we could remove that?